tests/test_ensembles.py

```diff
@@ -25,13 +25,18 @@
 from pkg_resources import parse_version
 from scipy import __version__ as __scipy_version__
 from scipy.stats.mstats import mquantiles
-from sklearn import __version__ as __sklearn_version__
 
 from xclim import ensembles
 from xclim.indices.stats import get_dist
 from xclim.testing.helpers import TESTDATA_BRANCH
 
 
+# sklearn's KMeans doesn't accept the standard numpy Generator, so we create a special fixture for these tests
+@pytest.fixture
+def random_state():
+    return np.random.RandomState(seed=list(map(ord, "𝕽𝔞𝖓𝔡𝖔𝔪")))
```

Comment:
Maybe! My guess here was to put the fixture close to the affected tests (same file), but at the top of the file so that all "conf" stuff is at the same place. (even though there isn't any other conf in this module)

Reply:
Placement within this file works for me. Maybe add another comment here mentioning that this functionality may be removed in a future version of numpy?